Mathlib/Tactic/ToAdditive/Frontend.lean

```diff
@@ -692,23 +693,39 @@ def reorderLambda (reorder : List (List Nat) := []) (src : Expr) : MetaM Expr :=
   lambdaTelescope src fun xs e => do
     mkLambdaFVars (xs.permute! reorder) e
 
+/-- Unfold auxlemmas in the type and value. -/
+def declUnfoldAuxLemmas (decl : ConstantInfo) : MetaM ConstantInfo := do
+  let mut decl := decl
+  decl := decl.updateType <| ← unfoldAuxLemmas decl.type
+  if let some v := decl.value? then
+    trace[to_additive] "value before unfold:{indentExpr v}"
```

Comment:
```suggestion
    trace[to_additive_detail] "value before unfold:{indentExpr v}"
```
I believe these traces are only for debugging, and rarely useful for users, right?

Also 2 lines below.